Project.toml

```diff
@@ -0,0 +1,37 @@
+name = "SyntheticGrids"
+uuid = "34f115b4-9156-5266-a33d-ca359e631fb8"
+license = "MIT"
+author = "Invenia Technical Computing"
+version = "0.1.0"
+
+[deps]
+AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Geodesy = "0ef565a4-170c-5f04-8de2-149903a85f3d"
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
+PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+[compat]
+AutoHashEquals = "^0.2.0"
+CSV = "^0.5.9"
+Conda = "^1.3.0"
+Geodesy = "^0.5.0"
+HDF5 = "^0.12.0"
+JSON = "^0.21.0"
+Missings = "^0.4.0"
```

Comment:
Missing support is included in Base as of Julia 1.0

Reply:
You should also remove this from `src/SyntheticGrids.jl` and `[deps]` in the `Project.toml`